src/Magento/FunctionalTestingFramework/Extension/TestContextExtension.php

```diff
@@ -173,7 +174,17 @@ public function beforeStep(\Codeception\Event\StepEvent $e)
      */
     public function afterStep(\Codeception\Event\StepEvent $e)
     {
-        ErrorLogger::getInstance()->logErrors($this->getDriver(), $e);
+        if (getenv('ENABLE_BROWSER_LOG')) {
+            $browserLog = $this->getDriver()->webDriver->manage()->getLog("browser");
+            foreach (explode(',', getenv('BROWSER_LOG_BLACKLIST')) as $source) {
+                $browserLog = BrowserLogUtil::filterLogsOfType($browserLog, $source);
+            }
```

Comment:
Can line 177 ~ 181 be moved to BrowserLogUtil class? The same logic is in BrowserLogUtil.

Reply:
Didn't watch this, I'll see if I can move it

Reply:
I did it the other way around, I removed redundant logic from `BrowserLogUtil::logErrors()`. This kind of log getting should be done by the consumer of the Util, not by the util itself in my opinion.